### ISSUE TITLE
.pryv directory does not exist at first start up

### ISSUE THREAD

**Issue by mboulhaj** [leopoldhoudin/pryv#3]
**Version**: 1.0.0a1

**Description**
When starting the server for the first time after installation, I get an error because `~/.pryv` directory does not exist.

Yet, manually creating the `~/.pryv` directory does solve the problem.

Maybe you should add a guard when creating the directories at first start-up?

**Stacktrace**:
```commandline
...
Traceback (most recent call last):
  File "/usr/local/bin/pryv", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/pryv/cli.py", line 103, in start
    from pryv.app import app
  File "/usr/local/lib/python2.7/site-packages/pryv/app.py", line 11, in <module>
    from pryv.api import mount as mount_api
  File "/usr/local/lib/python2.7/site-packages/pryv/api/__init__.py", line 7, in <module>
    from pryv.api.simple import controller as simple_controller
  File "/usr/local/lib/python2.7/site-packages/pryv/api/simple.py", line 16, in <module>
    from pryv.core import find_all_projects, find_project, create_project, parse_metadata
  File "/usr/local/lib/python2.7/site-packages/pryv/core/__init__.py", line 8, in <module>
    from pryv.core.package import Package, \
  File "/usr/local/lib/python2.7/site-packages/pryv/core/package.py", line 15, in <module>
    packages_proxy = ensure_directory(config.core.packages_path)
  File "/usr/local/lib/python2.7/site-packages/pryv/proxy/__init__.py", line 60, in ensure_directory
    fs_proxy.make_directory()
  File "/usr/local/lib/python2.7/site-packages/pryv/proxy/base.py", line 38, in decorator
    return func(instance, join_path(*paths), **kwargs)
  File "/usr/local/lib/python2.7/site-packages/pryv/proxy/base.py", line 458, in make_directory
    self._make_directory(path)
  File "/usr/local/lib/python2.7/site-packages/pryv/proxy/local.py", line 34, in decorator
    if path else func(instance, instance.path, *args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/pryv/proxy/local.py", line 133, in _make_directory
    os.mkdir(path)
OSError: [Errno 2] No such file or directory: '/Users/maryame/.pryv/packages'
```

**Comment by leopoldhoudin** [comment]
All right, I'll have a look at it.

It seems that `proxy.ensure_directory` does not ensure that much that the directory exists...

**Comment by leopoldhoudin** [comment]
Closed by #4
